examples/data-objects/badge/src/BadgeModel.tsx

```diff
@@ -59,10 +72,13 @@ export class Badge extends DataObject implements IBadgeModel, IFluidHTMLView {
      * object refs as props to the React component.
      */
     protected async hasInitialized() {
-        [this.currentCell, this.optionsMap, this.historySequence] = await Promise.all([
-            this.root.get<IFluidHandle<SharedCell>>(this.currentId).get(),
-            this.root.get<IFluidHandle<SharedMap>>(this.optionsId).get(),
-            this.root.get<IFluidHandle<SharedObjectSequence<IBadgeHistory>>>(this.historyId).get(),
+        [this._currentCell, this._optionsMap, this._historySequence] = await Promise.all([
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            this.root.get<IFluidHandle<SharedCell>>(this.currentId)!.get(),
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            this.root.get<IFluidHandle<SharedMap>>(this.optionsId)!.get(),
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Maybe it's worth turning this lint rule off? Seems like it's making you say "yes I really mean to say I really mean to say it's not null"?

Reply:
Yeah.  For this work, I try to make the least amount of code change as possible to enable `strictNullCheck` so new problem doesn't creep in.

In this particular place, I did add accessor to throw if it is not initialized, so I can switch to `?.` instead.

Reply:
There are places that still require to use null assertion, because condition to covariant with other check, or hidden in function calls.    It might be worth turning it off if that happens a lot.